examples/cpp-tracing/compiled-in/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.2"
+services:
+  dd-opentracing-cpp-example:
+    build:
+      context: ./
+    depends_on:
+      - dd-agent
+  dd-agent:
+    container_name: dd-agent
+    volumes:
+      - '/var/run/docker.sock:/var/run/docker.sock:ro'
+      - '/proc/:/host/proc/:ro'
+      - '/sys/fs/cgroup/:/host/sys/fs/cgroup:ro'
+    environment:
+      - 'DD_APM_ENABLED=true'
+      - DD_API_KEY=${DD_API_KEY}
```

Comment:
You get the same effect by just putting `- DD_API_KEY` (it'll inherit the value from the shell running docker-compose).